_inc/client/plans/single-product-backup.jsx

```diff
@@ -61,24 +63,54 @@ export function SingleProductBackup( { products, upgradeLinks } ) {
 	);
 }
 
-function SingleProductBackupHeader( { currencySymbol, backupPlanPrices } ) {
+function SingleProductBackupHeader( { currencySymbol, currencyCode, backupPlanPrices } ) {
 	return (
 		<div className="single-product-backup__header-container">
 			<h3 className="single-product-backup__header-title">{ __( 'Jetpack Backup' ) }</h3>
-			<PlanPriceDisplay currencySymbol={ currencySymbol } backupPlanPrices={ backupPlanPrices } />
+			<PlanPriceDisplay
+				currencySymbol={ currencySymbol }
+				currencyCode={ currencyCode }
+				backupPlanPrices={ backupPlanPrices }
+			/>
 		</div>
 	);
 }
 
-export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
+function formatCurrencyNumber( number, code ) {
```

Comment:
Do we really need to copy this over from `@automattic/format-currency`? If we imported it at the first place like:

```js
import formatCurrency from '@automattic/format-currency';
```

then we could use it as noted in the other comment.